Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:2.7.12
+EXPOSE 5000
+VOLUME /root/.octoprint
+
+RUN  wget -O ffmpeg.tar.xz https://johnvansickle.com/ffmpeg/releases/ffmpeg-release-32bit-static.tar.xz && mkdir -p /opt/ffmpeg && tar xvf ffmpeg.tar.xz -C /opt/ffmpeg --strip-components=1
+
+ADD ./ /opt/octoprint/
+RUN cd /opt/octoprint && python setup.py install
+
+CMD ["octoprint", "--iknowwhatimdoing"]
```

Comment:
Do I understand it correctly that this setup is basically enforcing to run OctoPrint as root and that's why that parameter is required? This is a bad idea then. Container or not, a user space program should never be run as root, and that option only exists for very very specific usecases where this can absolutely not be avoided - it should never be something set by default and really only be an individual user's/admin's choice, ideally a user/admin who indeed knows what they are doing in running that server as root. To be able to merge this we'll need to find a way around this.